src/utils/vc/credentials.js

```diff
@@ -270,7 +302,73 @@ export async function issueCredential(keyDoc, credential, compactProof = true, d
   // Ensure credential is valid
   checkCredential(cred);
 
-  // Sign and return the credential
+  // Should use JWT format?
+  if (useJWT) {
```

Comment:
could we pull the useJWT code out into helper functions that we can write tests for more simply? I always get nervous with `if` statements in large blocks of code

Reply:
its already split into helpers really:
```
    // Format to VC JWT spec
    const [vcJwtHeader, vcJwtPayload] = formatToJWTPayload(keyDoc, cred);

    // Get suite from keyDoc parameter
    const jwtOpts = { detached: false, header: vcJwtHeader };
    const suite = await getSuiteFromKeyDoc(keyDoc, useProofValue, jwtOpts);
    return signJWS(suite.signer || suite, suite.alg, jwtOpts, vcJwtPayload);
```

dont think this would be useful outside of this scope